pom.xml

```diff
@@ -23,6 +23,7 @@
         <familie.eksterne-kontrakter.saksstatistikk-ef>2.0_20230214104704_706e9c0</familie.eksterne-kontrakter.saksstatistikk-ef>
         <familie.eksterne-kontrakter.arbeidsoppfolging>2.0_20230214104704_706e9c0</familie.eksterne-kontrakter.arbeidsoppfolging>
         <prosessering.version>2.20230807154047_d770f01</prosessering.version>
+        <utbetalingsgenerator.version>1.0_20230921152358_e73110f</utbetalingsgenerator.version>
```

Comment:
Burde vi gå for nyeste versjon med en gang, eller skal vi sterte her. 

Helt med på å starte med denne hvis det er enklere. 

  <version>1.0_20230927085000_900743f</version>

Reply:
Bah, så ikke at det har kommet enda en ny oppdatering...

Reply:
Men jeg brukte jo 5 dager på den forrige bumpen